Aptoide/aptoide-client-v8
# [APP-2349] - Add AptoideGames toolbar logo and dev launcher icon

**What does this PR do?**

Replaces the GamesHub toolbar logo with the Aptoide Games one and fixes the padding since we now have the notification bell as well. The needed colors were also added.

Adds the launcher icon for both dev and prod. 

It also changes the theme's background color and forces the theme to be always darkMode.

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AptoideGamesToolbarLogo.kt
- [ ] Color.kt
- [ ] AppGamesToolBar.kt
- [ ] All the res files added as well

**How should this be manually tested?**

See if the launcher icon is well fit for all resolutions and if the toolbar logo is correctly fit on the top, with the correct colors. Test both dev and prod to see the icons.

  Flow on how to test this or QA Tickets related to this use-case: [APP-2349](https://aptoide.atlassian.net/browse/APP-2349)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2349](https://aptoide.atlassian.net/browse/APP-2349)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 


**Code Review Checklist**


[APP-2349]: https://aptoide.atlassian.net/browse/APP-2349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2349]: https://aptoide.atlassian.net/browse/APP-2349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
